src/controllers/carts.js

```diff
@@ -0,0 +1,21 @@
+const cartService = require('../services/cart');
+
+const showCart = async (req, res) => {
+  try {
+    const cart = await cartService.findCartbyUserId(req.body);
+    res.status(200).send(cart);
+  } catch (err) {
+    res.status(500).send({ error: 'Something went wrong:\n' + err.message });
+  }
+};
+
+const addProductToCart = async (req, res) => {
+    try {
+      const cart = await cartService.addProductToCart(req.body);
```

Comment:
same here, we should add the user.id from the JWT , maybe we can add a comment like
// TODO send the id from req.user and not take it from the body

Reply:
ok, added the comment